build.cake

```diff
@@ -194,17 +194,17 @@ Task("Preview")
 Task("Debug")
     .Does(() =>
     {
-        StartProcess("../Wyam/src/clients/Wyam/bin/Debug/wyam.exe",
-            "-a \"../Wyam/src/**/bin/Debug/**/*.dll\" -r \"docs -i\" -t \"../Wyam/themes/Docs/Samson\" -p");
+        StartProcess("../Wyam/src/clients/Wyam/bin/Debug/net462/wyam.exe",
+            "-a \"../Wyam/tests/integration/Wyam.Examples.Tests/bin/Debug/net462/**/*.dll\" -r \"docs -i\" -t \"../Wyam/themes/Docs/Samson\" -p");
```

Comment:
The paths here had to change since the client app now uses an SDK-style project file which puts binaries under a target framework subdirectory. Figured this was loosely related to bumping the version so I left it in here instead of adding another PR. I'm probably the only one who uses this build target anyway.